src/UseCases/Embed/EmbedRequestBuilder.php

```diff
@@ -25,7 +25,7 @@ public static function argumentsToRequest( array $arguments, bool $showEditButto
 		return new EmbedRequest(
 			fileUrl: $arguments[0],
 			language: is_string( $language ) ? $language : null,
-			showLineNumbers: is_bool( $line ) ? $line : null,
+			showLineNumbers: is_bool( $lineNumbers ) ? $lineNumbers : null,
 			showSpecificLines: is_string( $specificLines ) ? $specificLines : null,
```

Comment:
Shouldn't we change all variables, remove 'specific', and add 'show' (where applicable) that will make it consistent across the code?

Reply:
That would be a bit cleaner but is not a blocker for the release